### DISCUSSION TITLE
How to extract "Percent routed", "Suction head", "Conductivity", and "Initial deficit" for a subcatchment using PySWMM?

### DISCUSSION THREAD

**Discussion by Freestyleyang** [pyswmm/pyswmm#494] [Q&A]
Hi There,

Thank you for your great work! I am new to PySWMM and I am trying to extract the parameters "Percent routed", "Suction head", "Conductivity", and "Initial deficit" for a subcatchment. 

However, according to the PySWMM document, they are not available as "[Attributes](http://pyswmm.github.io/pyswmm/reference/api/pyswmm.subcatchments.Subcatchment.html)" for a subcatchment. Am I missing any information? Thank you.

**Comment by bemcdonnell** [comment]
@Freestyleyang, at the moment we have not exposed accessors to that information just yet.  Your options are to use `swmmio` or `swmm-api` or write your own parser to extract those parameters / results.  I'd be happy to provide some additional guidance after you review those options.  This project works directly with `swmmio`, if that helps you!

**Reply by Freestyleyang** [reply]
@bemcdonnell Thank you for your help and I will look into `swmmio`. It would be great if `PySWMM` can handle these parameters in the future version.
